.github/workflows/nos.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.8, 3.9, "3.10", "3.11"]
```

Comment:
please include 3.6 and 3.7 too

Reply:
Done.

Reply:
As I mentioned earlier, 3.6 is not available anymore in GH workflows as it seems: https://github.com/elcorto/gita/actions/runs/4047932391

Reply:
I just changed ubuntu-latest to ubuntu-20.04. Now python3.6 is tested

Reply:
Nice, haven't thought of that. Thanks.